Researcher page updates

abeconnelly opened the pull request:
- Added Q/A about where to find BAM/SAM/FASTQ/etc. files in researcher.md.
- Added some more pointers to Tapestry to search for data directly there instead of first pointing them to untap in researcher.md
- Added pointer to contact form in researcher.md for further questions (points to the PersonalGenomes.org contact form).
- Minor formatting and spelling changes all around (including the participant.md file).

abeconnelly committed:
cleanup and researcher data Q/A

* taking out some spurious spaces from participant.md
* added some more pointers to Tapestry for searching directly.
* added a question relating to BAM/SAM/FASTQ/etc. file formats.
* gave a pointer to the contact form for other questions.

abeconnelly committed:
Made "Where can I get..." answer more concise

abeconnelly committed:
spelling mistakes

abeconnelly committed:
formatting

abeconnelly committed:
more concise blog post pointer

madprime commented on researcher.md:
```diff
@@ -4,9 +4,25 @@
+
+The Harvard Personal Genome Project participant page, located at [my.pgp-hms.org](https://my.pgp-hms.org), holds information on participant data.
+Each Harvard Personal Genome Project participant has a profile page listing their available information.
+For example, the [profile participant for hu43860C](https://my.pgp-hms.org/profile/hu43860C) has medical information,
```
Maybe also note that there's JSON-format data for profiles that facilitates automated retrieval of data from profile pages. For example: https://my.pgp-hms.org/profile/hu43860C.json

madprime commented on researcher.md:
```diff
@@ -16,4 +32,23 @@ The spreadsheets and SQLite3 database only represent most of the data provided b
+
+Most genomic data available through the Harvard Personal Genome Project site is only available in the format provided to us by Complete Genomics (CGI) and would need to be converted.
+
+CGI gives us unaligned read information, variant evidence files and CGI-Var files, among others.
```
I believe that, for a very large fraction of our genomes, do not have "unaligned read information". As I understand it, we elected to not receive the "MAP" directories in data sets in exchange for a lower price. Description of MAP directory here - http://www.completegenomics.com/documents/DataFileFormats_Standard_Pipeline_2.5.pdf

abeconnelly committed:
adding '.json' and took out line

CGI didn't always provide us with MAP information (unaligned? read
information).  I just took it out.

Adding a .json extension for most pages results in JSON output and
I added that information in along with a note about how to
produce JSON output for the GET-Evidence pages.

abeconnelly committed:
CC0 license Q/A

it's redundant because it's located in the README.md proper but I think
it's good to have it here as well.

abeconnelly committed:
Merge branch 'master' of https://github.com/abeconnelly/FAQ

Conflicts:
	researcher.md

abeconnelly committed:
remove CGI unaligned read line

abeconnelly commented:
- Added a Q/A about the `.json` extension (also mentioned the GET-Evidence report)
- Took out line about receiving unaligned reads from CGI
- Added in CC0 license (redundant but good to have here as well)

celiafw commented:
Thank you Abram, that is just fantastic and will be super helpful going
forward.

Celia

On Mon, Dec 7, 2015 at 11:18 AM, Abram Connelly notifications@github.com
wrote:

> - Added a Q/A about the .json extension (also mentioned the
>   GET-Evidence report)
> - Took out line about receiving unaligned reads from CGI
> - Added in CC0 license (redundant but good to have here as well)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/PGPHarvard/FAQ/pull/8#issuecomment-162629608.

nouyang-curoverse commented:
I believe have merged these changes into the wiki, notably I changed the support email to the PGP Harvard form instead of personalgenomes.org contact form (seems more appropriate)? Thanks all!
